keanu-project/src/main/java/io/improbable/keanu/tensor/intgr/ScalarIntegerTensor.java

```diff
@@ -295,7 +297,7 @@ public IntegerTensor powInPlace(IntegerTensor exponent) {
 
     @Override
     public IntegerTensor powInPlace(int exponent) {
-        value = (int) Math.pow(value, exponent);
+        value = IntMath.pow(value, exponent);
```

Comment:
I think the emphasis is on correctness, but it's also efficient.

From the javadoc:

_"Even if the result overflows, it will be equal to BigInteger.valueOf(b).pow(k).intValue(). This implementation runs in O(log k) time."_